pkg/network/netlink/conntracker.go

```diff
@@ -42,10 +45,22 @@ type connKey struct {
 	transport network.ConnectionType
 }
 
+type translationEntry struct {
+	*network.IPTranslation
+	orphan *list.Element
+}
+
+type orphanEntry struct {
+	key connKey
+	ttl time.Time
```

Comment:
can we call this `expires` or something, since it is a point in time, rather than a TTL/duration?

Reply:
Sure